content/docs/introduction/comparison.md

```diff
@@ -160,6 +197,95 @@ simple aggregations via its API.
 means that it is easy to scale OpenTSDB horizontally, but you have to accept
 the overall complexity of running a Hadoop/HBase cluster from the beginning.
 
-Again, as in the case of InfluxDB, Prometheus will be simpler to run initially,
-but will require explicit sharding once the capacity of a single node is
-exceeded.
+Prometheus will be simpler to run initially, but will require explicit sharding
+once the capacity of a single node is exceeded.
+
+### Summary
+
+Prometheus offers a much richer query language, can handle higher cardinality
+metrics and forms part of a complete monitoring system. If you're already
+running Hadoop and value long term storage over these benefits, OpenTSDB is a
+good choice.
+
+
+## Prometheus vs. Nagios
+
+[Nagios](https://www.nagios.org/) is a monitoring system that originated in the
+90s as NetSaint.
+
+### Scope
+
+Nagios is primarily about alerting based on the exit codes of scripts. These are called "checks".
+There is silencing of individual alerts, however no grouping, routing or deduplication.
+
+There are a variety of plugins. For example, piping the few kilobytes of
+perfData plugins are allowed to return [to a time series database such as
+Graphite](https://github.com/shawn-sterling/graphios) or using NRPE to [run
+checks on remote
+machines](https://exchange.nagios.org/directory/Addons/Monitoring-Agents/NRPE--2D-Nagios-Remote-Plugin-Executor/details).
+
+### Data model
+
+Nagios is host-based. Each host can have one or more services, which has one check.
+
+There is no notion of labels or a query language.
+
+### Storage
+
+Nagios has no storage per-se, beyond the current check state.
+There are plugins which can store data such as [for visualisation](https://docs.pnp4nagios.org/).
+
+### Architecture
+
+Nagios servers are standalone. All configuration of checks is via file.
+
+### Summary
+
+Nagios is suitable for basic monitoring of small and/or static systems where
+blackbox probing is sufficient.
```

Comment:
Not sure about this. It doesn't have it inherently, but there's plenty of common ways it is done which I've referred to above.